plugin-build/modulegraph/src/main/kotlin/dev/iurysouza/modulegraph/gradle/ModuleGraphExtension.kt

```diff
@@ -44,6 +45,13 @@ open class ModuleGraphExtension @Inject constructor(project: Project) {
      */
     val linkText: Property<LinkText> = objects.property(LinkText::class.java)
 
+    /**
+     * List of configuration name which should be ignored.
```

Comment:
```suggestion
     * Configuration names to be ignored.
```

Reply:
As https://github.com/iurysza/module-graph/pull/25#discussion_r1540885443,  https://github.com/iurysza/module-graph/pull/25#discussion_r1540969395 says, 
Maybe it should be "List of configurations to be ignored" ??

Reply:
fixed https://github.com/iurysza/module-graph/pull/25/commits/73b5e936e23403336309bc043e887c3c0bd3aa7d